app/channels/game_data_channel.rb

```diff
@@ -1,10 +1,36 @@
 class GameDataChannel < ApplicationCable::Channel
+  on_subscribe :welcome_player
+
   def subscribed
-    player = Player.includes(:game, :user).find_by(token: connection.current_player.token)
-    stream_from "game_#{player.game.game_key}"
+    # ActionCable doesn't give access to params[] here if mounted in application.rb
+    @player = Player.includes(:game, :user).find_by(token: connection.current_player.token)
+    stream_from "game_#{@player.game.game_key}"
```

Comment:
Will this now be the channel name?

Reply:
Not much change here... just made the `player` into an instance variable, so I had to change the reference when interpolating the game key into the channel name.